c/tskit/core.c

```diff
@@ -198,6 +198,15 @@ tsk_strerror_internal(int err)
         case TSK_ERR_PROVENANCE_OUT_OF_BOUNDS:
             ret = "Provenance out of bounds";
             break;
+        case TSK_ERR_NODE_TIME_NONFINITE:
```

Comment:
We're skipping migrations here, which have left, right and time float values IIRC. It's a bit painful to add two more errors just for this. Maybe `TSK_ERR_TIME_NONFINITE` and `TSK_ERR_COORDS_NONFINITE` and then these can be used in the migrations. Or, if you prefer to add two additional errors then that's fine too.

Reply:
I've added migrations. The error is now `TSK_ERR_GENOME_COORDS_NONFINITE` so that we don't get confused with other sorts of coordinates (e.g. lat/long), and to make the location error completely clear and more general in the same way, I've changed `TSK_ERR_INDIVIDUAL_LOCATION_NONFINITE` to `TSK_ERR_SPATIAL_LOCATION_NONFINITE`.

https://github.com/tskit-dev/tskit/blob/7656bedfd29e46f450ed99f9fe17494e805d39ed/c/tskit/tables.c#L5759 has a `TODO it's a bit misleading to use the edge-specific errors here.` - but it seems like we are now happy using errors like `TSK_ERR_LEFT_LESS_ZERO` for genome coords in general. So should we remove the TODO, or maybe rename the error to something more verbose like `TSK_ERR_LEFT_GENOME_COORD_LESS_ZERO`?